CMakeLists.txt

```diff
@@ -1209,7 +1209,7 @@ if(ZLIB_ENABLE_TESTS)
     add_test(NAME makefixed
         COMMAND ${CMAKE_COMMAND}
         "-DCOMMAND=${MAKEFIXED_COMMAND}"
-        -DOUTPUT=${CMAKE_CURRENT_BINARY_DIR}/inffixed_tbl._h
+        -DOUTPUT=${CMAKE_CURRENT_BINARY_DIR}/Testing/Temporary/inffixed_tbl._h
```

Comment:
This doesn't make sense at all... `CMAKE_CURRENT_BINARY_DIR` is never supposed to be read-only.

Reply:
Good point, perhaps if `CMAKE_CURRENT_BINARY_DIR=CMAKE_CURRENT_SOURCE_DIR`?

Reply:
Obviously we need to check if we are doing out-of-tree build... See my previous comment...